GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -15,8 +15,14 @@
  */
 
 import Foundation
+// This check is needed to ensure that we import the correct dependency for both SPM and CocoaPods
```

Comment:
Might add something along the lines of "CocoaPods doesn't define separate Clang modules for subspecs."